.spelling

```diff
@@ -0,0 +1,724 @@
+# markdown-spellcheck spelling configuration file
+# Format - lines beginning # are comments
+# global dictionary is at the start, file overrides afterwards
+# one word per line, to define a file override use ' - filename'
+# where filename is relative to this configuration file
+PowerShell
```

Comment:
Probably, but in practice, I find it pretty usable to just ctrl+f the file if I have to delete a work or so.

Moreover, since this file is created and managed by the spell-checker cli, the sorting have to be done manually. I've just submitted a feature request at https://github.com/lukeapage/node-markdown-spellcheck/issues/83